MVC/application/model/userAccount.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+  class userAccount extends database
```

Comment:
Since we are extending the database class here, let's explain the functionality of this class

Reply:
In `database` class I write the connection code with database.

Reply:
Fine. While adding comment add the same